content/containers.md

```diff
@@ -30,26 +30,37 @@ Ubuntu : <kbd>CTRL</kbd>+<kbd>ALT</kbd>+<kbd>T</kbd>
 Install the software with this command in the terminal.
 
 ```bash
-sudo apt install lxd lxd-client
+sudo apt install snapd
+snap install lxd
 ```
 
 ## Setup
 
 Tell LXD what storage and network is needed with this command:
 
 ```bash
-sudo lxd init
+lxd init
```

Comment:
I had to add my user to the `lxd` group and reboot before these commands would work (should be doable with `sudo usermod -aG lxd $USER`.) Before that, the first one (init) didn't complete properly and the second one (launch) just gave me a "permission denied" error.

Reply:
Should be fixed in the newest commit.

Reply:
The reboot that I mentioned was not optional, your updated instruction still don't work. I'll add it myself.